features/cli/oc_registry.feature

```diff
@@ -0,0 +1,40 @@
+Feature: oc registry command scenarios
+
+  # @author wzheng@redhat.com
+  # @case_id OCP-21926
+  Scenario: Check function of oc registry command
+    Given I have a project
+    When I run the :registry_info client command with:
+      | internal | true |
+    Then the step should succeed
+    And the output should contain:
+      | image-registry.openshift-image-registry.svc:5000 |
```

Comment:
@wzheng1 does this output changes depending on the cluster configuration?  Like in-cluster registry?

Reply:
No, all 4.x cluster has the same name like this.

Reply:
@wzheng1 If this will report error when the route for default regtistry is enabled?

Reply:
No, that's why I use internal=true option